library/ziti.c

```diff
@@ -680,8 +678,10 @@ const char *ziti_conn_source_identity(ziti_connection conn) {
 
 
 void ziti_send_event(ziti_context ztx, const ziti_event_t *e) {
-    if ((ztx->opts->events & e->type) && ztx->opts->event_cb) {
-        ztx->opts->event_cb(ztx, e);
+    if (ztx->enabled || e->type == ZitiContextEvent) {
```

Comment:
only ZitiContextEvent?

Reply:
once ztx is disabled (via ziti_shutdown()) app would not expect other events